internal/services/web/app_service_certificate_binding_resource.go

```diff
@@ -126,7 +123,7 @@ func resourceAppServiceCertificateBindingCreate(d *pluginsdk.ResourceData, meta
 
 	props := binding.HostNameBindingProperties
 	if props != nil {
-		if props.Thumbprint != nil && *props.Thumbprint == *thumbprint {
+		if props.SslState != "" || props.VirtualIP != nil {
```

Comment:
can't SSL State be disabled?

Reply:
Yes, but it seems that only un-configured bindings are empty for this property. If it has a setting, it is in some way being managed and I think should be imported.